ntoskrnl/ke/amd64/mp.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * FILE:        ntoskrnl/ke/amd64/mp.c
+ * PURPOSE:     Source file to hold multiprocessor functions
+ * PROGRAMMERS:  Copyright 2021 Justin Miller <justinmiller100@gmail.com>
+ */
+
+/* INCLUDES *****************************************************************/
+
+#include <ntoskrnl.h>
+#define NDEBUG
+#include <debug.h>
+
+/* GLOBALS *******************************************************************/
+
+#define PHYSICAL_ADDRESS  LARGE_INTEGER
```

Comment:
oh not this in amd64 please 😭

Reply:
Huh? Is there a better alternative?

Reply:
Using ExAllocatePoolWithTag, as Timo pointed out ;-)